README.md

```diff
@@ -70,15 +70,70 @@ and to stop all the processes you can do
 stopObserveAll
 ``` -->
 
-# How to package for deployment
+# Deployment
 
-If you haven't already, create a symlink `jre` in the project directory that points to the Linux JRE you want bundled with the deployment.
+Deployment is done via Docker images.
 
-To deploy, run in `sbt`:
+## Staging
+
+Make sure you have both `docker` and `heroku` CLIs installed and working.
+
+If you haven't already, run
+
+```
+heroku login
+heroku container:login
+```
+
+This will give your system access to Heroku's Docker registry.
```

Comment:
What do we need to use heroku's registry from inside the gemini network?

Reply:
We will probably use Dockerhub for testing/production deployment, not Heroku's registry. I think we just need access to `hub.docker.com:443`, but I'll double check.